spec/integration/inventory_spec.rb

```diff
@@ -25,24 +25,46 @@ module Hawkular::Inventory::RSpec
     end
   end
 
-  describe 'Inventory', vcr: { decode_compressed_response: true } do
+  describe 'Inventory' do
     before(:all) do
       @creds = {
         username: 'jdoe',
         password: 'password'
       }
       @client = Hawkular::Inventory::InventoryClient.create(credentials: @creds)
-      @state = {
-        hostname: 'localhost.localdomain',
-        feed: nil
-      }
+      options = { decode_compressed_response: true }
+      options[:record] = :all if ENV['VCR_UPDATE'] == '1'
+      VCR.use_cassette('Inventory/Helpers/get_feeds', options) do
+        feeds = @client.list_feeds
+        @state = {
+          hostname: 'dhcp-10-40-1-43.brq.redhat.com',
```

Comment:
yeah, the `hostname` isn't the ip/host of the hawkular into which the vcr is connecting, but the value that is expected in the JSON. I think it's stored in the configuration of wildfly resource. 

The tests currently run always against localhost (https://github.com/Jiri-Kremser/hawkular-client-ruby/blob/0.3.0/spec/integration/inventory_spec.rb#L10)